Source/WebCore/platform/graphics/cg/ImageBufferIOSurfaceBackend.cpp

```diff
@@ -249,6 +254,15 @@ void ImageBufferIOSurfaceBackend::setVolatilityState(VolatilityState volatilityS
     m_volatilityState = volatilityState;
 }
 
+void ImageBufferIOSurfaceBackend::forcePurgeForTesting()
+{
+    ASSERT(!m_context);
+    CGContextClearRect(ensurePlatformContext(), FloatRect(FloatPoint::zero(), size()));
```

Comment:
The contract seems to be that setVolatile has been called. So you cannot modify a volatile surface.